wrapper-cpp/modules/c_meeting_chat_interface.cpp

```
@@ -18,6 +18,59 @@ extern "C" ZOOMSDK::IChatMsgInfo *meeting_chat_build(
+    void onChatMsgDeleteNotification(const zchar_t* msgID, ZOOMSDK::SDKChatMessageDeleteType deleteBy) override { (void)msgID; (void)deleteBy; }
+    void onChatMessageEditNotification(ZOOMSDK::IChatMsgInfo* chatMsg) override { (void)chatMsg; }
+    void onShareMeetingChatStatusChanged(bool isStart) override { (void)isStart; }
+    void onFileSendStart(ZOOMSDK::ISDKFileSender* sender) override { (void)sender; }
+    void onFileReceived(ZOOMSDK::ISDKFileReceiver* receiver) override { (void)receiver; }
+    void onFileTransferProgress(ZOOMSDK::SDKFileTransferInfo* info) override { (void)info; }
+
+private:
+    void *ptr_to_rust;
+};
+
+extern "C" ZOOMSDK::SDKError chat_set_event(ZOOMSDK::IMeetingChatController *controller, void *arc_ptr) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)